src/python/bot/startup/run_bot.py

```diff
@@ -189,6 +190,10 @@ def main():
 
 
 if __name__ == '__main__':
+  if sys.version_info.major == 3:
+    # TODO(ochang): Remove check once all migrated to Python 3.
+    multiprocessing.set_start_method('spawn')
```

Comment:
Will this impact OSS-Fuzz bots ? I was thinking on process_handler.get_process ? Thoughts ?

Reply:
I guess this will impact JS fuzzers there, but there shouldn't be negative consequences. Grey box fuzzers don't use multiprocessing.Process there.